lxd/storage/backend_lxd.go

```diff
@@ -2920,6 +2920,13 @@ func (b *lxdBackend) UpdateInstance(inst instance.Instance, newDesc string, newC
 			return fmt.Errorf(`Instance volume "volatile.uuid" property cannot be changed`)
 		}
 
+		if shared.IsFalseOrEmpty(changedConfig["security.shared"]) && volDBType == cluster.StoragePoolVolumeTypeVM {
```

Comment:
what do you think about adding a function that indicates if the volDBType is an instance volume (container or VM), and then preemptively use that function in places like this to apply to both VM and containers, even though we dont yet allow adding container's root disk to other containers?

Reply:
There's only one other spot it's used this way, but it's a reasonable idea. I went ahead and added it.

Reply:
I'll tuck that away in a branch and polish it off after this is merged.